### PR TITLE
p-AAA MIMO Fix

### LINKED ISSUE #2210: non-SISO data leads to shape error in `PAAAReductor`
```python
from pymor.reductors.aaa import PAAAReductor

sv = np.arange(10)
H = np.random.rand(10, 5, 1)
aaa = PAAAReductor(sv, H)
tf = aaa.reduce(tol=1e-3)
tf.eval_tf(0)
```
gives me
```
ValueError: shapes (1,9) and (9,1,5) not aligned: 9 (dim 1) != 1 (dim 1)
```

### PR DESCRIPTION
**Pull request by lbalicki**
Fixes #2210. For me it works for parametric/non-parametric and all MIMO scenarios.

### CODE REVIEW

**Comment by pmli** [comment]
> For me it works for parametric/non-parametric and all MIMO scenarios.

Could you add them as tests?

**Review by artpelling — APPROVED** [review]
Works!

**Comment by lbalicki** [comment]
I added some simple tests checking the different cases.

File: src/pymortests/reductors/paaa.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+# License: BSD 2-Clause License (https://opensource.org/licenses/BSD-2-Clause)
+
+import numpy as np
+import pytest
+
+from pymor.reductors.aaa import PAAAReductor
+
+pytestmark = pytest.mark.builtin
+
+np.random.seed(0)
```
Comment (pmli) [inline src/pymortests/reductors/paaa.py:12]: I'm not sure, but maybe this should be inside the function to fix the `sampling_values` and `samples`.

**Review by pmli — APPROVED** [review]

**Pull request merged**